wallet-api/src/Wallet/Emulator/Http.hs

```diff
@@ -52,26 +71,26 @@ data AppError
 wallets ::
      (MonadError ServantErr m, MonadReader State m, MonadIO m) => m [Wallet]
 wallets = do
-  var <- asks getWallets
+  var <- asks getState
   ws <- liftIO $ readTVarIO var
-  pure $ HM.keys ws
+  pure $ ws ^. walletStates . to Map.keys
```

Comment:
Why can't this just be `Map.keys $ _walletStates ws`?

Reply:
it can be but the lenses were already there so this was really an attempt at consistency

Reply:
Oh. I'd prefer using the more explicit approach even if other code may be muddy - `to Map.keys` is a bit silly.